jsonschema2pojo-gradle-plugin/example/android/app/build.gradle

```diff
@@ -29,11 +29,26 @@ android {
         }
     }
 
+    packagingOptions {
+        exclude 'META-INF/LICENSE.txt'
+    }
+
 }
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:21.0.3'
+    compile 'com.android.support:appcompat-v7:23.1.1'
+
+    // Required for @Generated annotation
+    compile 'org.glassfish:javax.annotation:10.0-b28'
```

Comment:
Android doesn't contain the full Java language and is missing @Generated annotation.
I've added it only to the Android example project.

It would be nice if annotationStyle=none would take care not to create @Generated annotation.  Or if there was another parameter to control if @Generated would be added to the generated class.